unity/Assets/Scripts/UnityMonoBehaviour/UnityStack.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Assets.Scripts.Interfaces;
+using Hexxle.TileSystem;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class UnityStack : MonoBehaviour
+{ 
+    public GameObject RedTemplate;
+    public GameObject GreenTemplate;
+    public GameObject BlueTemplate;
+    private TileStack tileStack = new TileStack();
```

Comment:
I'd prefer an Interface like ITileStack here, so we can keep some sort of contract of what this class is able to do in the future.

Reply:
I would argue against any form of interfaces. We do not need to bloat our work with unnecessary OOP-oriented features. The chances of us implementing a different stack in the future are, according to our project sheet, close to 0, so this seems like wasted effort.